sdk/forms/src/main/java/com/klaviyo/forms/BridgeMessage.kt

```diff
@@ -0,0 +1,163 @@
+package com.klaviyo.forms
```

Comment:
Renamed this to BridgeMessage, more in line with how our naming conventions have evolved. And I merged the Utils into this file as a companion factory method.